Tests/SonarScanner.MSBuild.Common.Test/Infrastructure/OutputCaptureScope.cs

```diff
@@ -64,6 +64,12 @@ public void AssertExpectedLastMessage(string expected)
         lastMessage.Should().Be(expected, "Expected message was not logged");
     }
 
+    public void AssertExpectedLastMessageRegex(string expected)
+    {
+        var lastMessage = GetLastMessage(outputWriter);
+        lastMessage.Should().MatchRegex(expected, "Expected message was not logged");
```

Comment:
Need to match the timestamp abstractly, as it does not have a concrete, static value.